### PR TITLE
Update dotnet new templates

### PR DESCRIPTION
**Pull request by jeromelaban**
## PR Type

What kind of change does this PR introduce?
- Bugfix

## What is the current behavior?

dotnet new templates are not behaving properly when used in the new VS and VSmac integrations.

## What is the new behavior?

Texts and options are adjusted.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: doc/articles/get-started-vsmac.md [hunk 1]
```diff
@@ -3,34 +3,52 @@
...
-## Modifying Existing Uno App
-
-1. Open project in Visual Studio for Mac
- ![new-project](Assets/quick-start/vs-mac-open-project.png)
-Once open, you should see your folder structure set up like this:
+* [.NET Core SDK 3.1](https://dotnet.microsoft.com/download/dotnet-core/thank-you/sdk-3.1.404-macos-x64-installer)
+* [**GTK+3**](https://formulae.brew.sh/formula/gtk+3) for running the Skia/GTK projects
+
+## Installing the dotnet new templates
+In order to create a new Uno Project, you'll need to open a terminal and run the following:
```
Comment (davidjohnoliver) [inline doc/articles/get-started-vsmac.md:13]: Link to the complete `dotnet new` doc? https://github.com/unoplatform/uno/blob/master/doc/articles/get-started-dotnet-new.md

File: doc/articles/get-started-vsmac.md [hunk 2]
```diff
@@ -3,34 +3,52 @@
...
+    - Right click to remove them from the solution.
+    - Note that you can create a `dotnet new` project without those two projects with:
+    ```bash
+    dotnet new unoapp -o MyProject --skia-wpf=false
+    ```
+1. You can now run on iOS, Android, and macOS by setting your startup project and running.
 ![startup-projects](Assets/quick-start/vs-mac-build.png)
    
-Note: You will not be able to build the UWP project on a Mac. All changes to this project must be made on Windows.
+Note: You will not be able to build the UWP and WPF projects on a Mac. All changes to this project must be made on Windows.
```
Comment (davidjohnoliver) [inline doc/articles/get-started-vsmac.md:43]: This is confusing, you've just told us to delete the WPF head. 

Also - why does the WPF head need to be deleted, but the UWP head not?

File: src/SolutionTemplate/Uno.ProjectTemplates.Dotnet/content/unoapp/.template.config/vs-2017.3.host.json [hunk 3]
```diff
@@ -64,14 +47,14 @@
...
       "id": "skia-wpf",
       "name": {
-        "text": "Enables the Skia/WPF platform support project"
+        "text": "Add support for Skia/WPF (Windows 7/10)"
       },
       "isVisible": "true"
     },
     {
-      "id": "skia-gtk",
+      "id": "UWP",
```
Comment (davidjohnoliver) [inline src/SolutionTemplate/Uno.ProjectTemplates.Dotnet/content/unoapp/.template.config/vs-2017.3.host.json:55]: 'skia-gtk' is intentionally removed?

File: doc/articles/get-started-vsmac.md (hunk 2 above)
Comment (jeromelaban) [reply doc/articles/get-started-vsmac.md:43]: Fair question, this section may need to be updated with the updated package I'm about to test (Where it's possible to remove items based on the IDE).

File: src/SolutionTemplate/Uno.ProjectTemplates.Dotnet/content/unoapp/.template.config/vs-2017.3.host.json (hunk 3 above)
Comment (jeromelaban) [reply src/SolutionTemplate/Uno.ProjectTemplates.Dotnet/content/unoapp/.template.config/vs-2017.3.host.json:55]: Not at all!! thanks for the catch :)

File: doc/articles/get-started-vsmac.md (hunk 2 above)
Comment (jeromelaban) [reply doc/articles/get-started-vsmac.md:43]: The problem here will be removed soon-ish one VS updates. I'll keep it as a heads up for people that want to still do windows dev.

File: src/SolutionTemplate/Uno.ProjectTemplates.Dotnet/content/unoapp/.template.config/vs-2017.3.host.json [hunk 4]
```diff
@@ -61,17 +44,24 @@
       },
       "isVisible": "true"
     },
+    {
+      "id": "skia-gtk",
+      "name": {
+        "text": "Enables the Skia/GTK platform support project"
```
Comment (davidjohnoliver) [inline src/SolutionTemplate/Uno.ProjectTemplates.Dotnet/content/unoapp/.template.config/vs-2017.3.host.json:50]:
```suggestion
        "text": "Add support for Skia/GTK (Linux)"
```

File: src/SolutionTemplate/Uno.ProjectTemplates.Dotnet/content/unolib-crossruntime/.template.config/vs-2017.3.host.json [hunk 5]
```diff
@@ -1,51 +1,17 @@
...
   "name": {
-    "text": "Uno WebAssembly App",
-    "id": "1060"
+    "text": "Cross-Runtime Library",
   },
   "description": {
-    "text": "A project template for creating rich Cross Platform app",
-    "package": "{0CD94836-1526-4E85-87D3-FB5274C5AFC9}",
-    "id": "1051"
+    "text": "A project template for creatin a Cross-Runtime Library",
```
Comment (davidjohnoliver) [inline src/SolutionTemplate/Uno.ProjectTemplates.Dotnet/content/unolib-crossruntime/.template.config/vs-2017.3.host.json:7]:
```suggestion
    "text": "A project template for creating a Cross-Runtime Library",
```

**Review by davidjohnoliver — APPROVED** [review]

**Review by tony981 — COMMENTED** [review]
Idk how to do this

**Pull request merged**